cmd/cleanup/runner.go

```diff
@@ -207,6 +208,32 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		}
 	}
 	r.logger.LogCtx(ctx, "message", "deleted release CR")
+
+	r.logger.LogCtx(ctx, "message", fmt.Sprintf("checking the namespace %#q deletion", r.flag.ClusterID))
```

Comment:
```suggestion
	r.logger.LogCtx(ctx, "message", fmt.Sprintf("deleting namespace %#q", r.flag.ClusterID))
```

Reply:
This CLI would not delete the cluster namespace but watch its deletion, but I think this is okay.